drivers/clk/clk_rpmsg.c

```diff
@@ -497,6 +497,10 @@ static int64_t clk_rpmsg_sendrecv(FAR struct rpmsg_endpoint *ept,
           ret = cookie.result;
         }
     }
+  else
+    {
+      rpmsg_release_tx_buffer(ept, msg);
```

Comment:
why not wrapper a common function to release holding buffer internally?

Reply:
OpenAMP is the external project, could you provide your suggestion to https://github.com/OpenAMP/open-amp/pulls? We can call the new function after the new function get merged.